### PR TITLE
placeholder confs

### PR DESCRIPTION
**Pull request by manoj-freyr**
enable build target and initial conf files

### CODE REVIEW

File: rvs/conf/nv48/gpup_single.conf [hunk 1]
```diff
@@ -0,0 +1,174 @@
+# ################################################################################
+# #
+# # Copyright (c) 2018-2022 Advanced Micro Devices, Inc. All rights reserved.
```
Comment (jkottiku) [inline rvs/conf/nv48/gpup_single.conf:3]: Update year to 2024 in all the new conf. files

**Review by jkottiku — CHANGES_REQUESTED** [review]

**Comment by jkottiku** [comment]
Do we need platform specific conf. files for rcqt, mem, pesm, gpup and peqt generic modules ?

**Comment by johnnynunez** [comment]
here we go! matrix cores coming to rdna4? 💯

File: rvs/conf/nv48/gpup_single.conf (hunk 1 above)
Comment (manoj-freyr) [reply rvs/conf/nv48/gpup_single.conf]: done

**Comment by manoj-freyr** [comment]
> Do we need platform specific conf. files for rcqt, mem, pesm, gpup and peqt - generic modules right ?

problem is with consistency. if we ask users to use specific for one set and generic for another set, as we have seen more confusion ensues. Will come up with a better way soon

File: rvs/conf/nv48/gst_single.conf [hunk 2]
```diff
@@ -0,0 +1,151 @@
+# ################################################################################
+# #
+# # Copyright (c) 2018-2023 Advanced Micro Devices, Inc. All rights reserved.
```
Comment (jkottiku) [inline rvs/conf/nv48/gst_single.conf:3]: Change year to 2024 as well

File: rvs/conf/nv48/gst_stress_3_hrs.conf [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# ################################################################################
+# #
+# # Copyright (c) 2018-2023 Advanced Micro Devices, Inc. All rights reserved.
```
Comment (jkottiku) [inline rvs/conf/nv48/gst_stress_3_hrs.conf:3]: Change year to 2024 as well

**Review by jkottiku — APPROVED** [review]

**Pull request closed without merging**